src/occa/internal/lang/type/pointer.cpp

```diff
@@ -2,18 +2,18 @@
 
 namespace occa {
   namespace lang {
-    pointer_t::pointer_t() {}
+    // pointer_t::pointer_t() {}
 
     pointer_t::pointer_t(const qualifiers_t &qualifiers_) :
       qualifiers(qualifiers_) {}
 
-    pointer_t::pointer_t(const pointer_t &other) :
-      qualifiers(other.qualifiers) {}
+    // pointer_t::pointer_t(const pointer_t &other) :
+    //   qualifiers(other.qualifiers) {}
 
-    pointer_t& pointer_t::operator = (const pointer_t &other) {
-      qualifiers = other.qualifiers;
-      return *this;
-    }
+    // pointer_t& pointer_t::operator = (const pointer_t &other) {
+    //   qualifiers = other.qualifiers;
+    //   return *this;
+    // }
```

Comment:
Why was this commented out? If we don't need it we can remove it

Reply:
It can be removed.